src/gui/accountsettings.cpp

```diff
@@ -202,6 +202,7 @@ void AccountSettings::slotOpenAccountWizard()
 void AccountSettings::slotToggleSignInState()
 {
     if (_accountState->isSignedOut()) {
+        _accountState->account()->resetRejectedCertificates();
```

Comment:
@ogoffart  I'm not sure I understand. Do you mean to say that "Log into all accounts" should not un-reject certificates? If so, then I agree. 

But clicking "log in" for one single account should be consistent no matter whether from the tray menu or the accountsettings. Therefore I suggest adding `resetRejectedCertificates` to the single-account case in `ownCloudGui::slotLogin`.